docs/overview.rst

```diff
@@ -29,23 +29,25 @@ You must access the Cloud Console by |region|.
 
 Available regions:
 
-+-------------------+-----------------------------------+
-| Region            | URL                               |
-+===================+===================================+
-| Azure East-US2    | `eastus2.azure.cratedb.cloud`_    |
-+-------------------+-----------------------------------+
-| Azure West-Europe | `westeurope.azure.cratedb.cloud`_ |
-+-------------------+-----------------------------------+
-| Bregenz           | `bregenz.a1.cratedb.cloud`_       |
-+-------------------+-----------------------------------+
++-------------------+-------------------------------------+
+| Region            | URL                                 |
++===================+=====================================+
+| AWS West Europe   | `eks1.eu-west-1.aws.cratedb.cloud`_ |
+| Azure East-US2    | `eastus2.azure.cratedb.cloud`_      |
++-------------------+-------------------------------------+
+| Azure West-Europe | `westeurope.azure.cratedb.cloud`_   |
++-------------------+-------------------------------------+
+| Bregenz           | `bregenz.a1.cratedb.cloud`_         |
++-------------------+-------------------------------------+
 
 Azure East-US2 and Azure West-Europe are managed by |Microsoft Azure|. The
-Bregenz region is managed by Crate.io and is located in Austria.
+Bregenz region is managed by Crate.io and is located in Austria. The AWS region
+is managed by AWS and is located in Ireland.
 
 Azure East-US2 is a good default region if you don't know which one to pick.
```

Comment:
We probably won't need this suggestion. All regions are equal and it's up to the user to decide which one to take.